src/Sulu/Component/Content/Repository/ContentRepository.php

```diff
@@ -186,7 +186,7 @@ public function findParentsWithSiblingsByUuid(
             ->orderBy($this->qomFactory->propertyValue('node', 'jcr:path'))
             ->where($this->qomFactory->childNode('node', $path));
 
-        while ($path !== $contentPath) {
+        while (PathHelper::getPathDepth($path) > PathHelper::getPathDepth($contentPath)) {
```

Comment:
yes and no (: this should be in the internal-link PR because there you can mix the webspaces and then the check has to be changed like this. i will move it to the internal-link PR.